deployment/human-connection/service-nginx.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: nitro-web
+  namespace: human-connection
+  labels:
+    human-connection.org/selector: deployment-human-connection-web
+spec:
+  ports:
+    - name: web
+      port: 3000
+      targetPort: 3000
+  selector:
+    human-connection.org/selector: nginx
```

Comment:
I myself don't know exactly what connects the service with the number of deployed pods from your deployment. But by comparing `deployment-web.yaml` and `service-web.yaml` it must be this `spec.selector` which has to match.